codec-http2/src/test/java/io/netty/handler/codec/http2/Http2ConnectionRoundtripTest.java

```diff
@@ -1207,6 +1223,15 @@ protected void initChannel(Channel ch) throws Exception {
                         .validateHeaders(false)
                         .build());
                 p.addLast(serverHandlerRef.get());
+                p.addLast(new ChannelInboundHandlerAdapter() {
+                    @Override
+                    public void userEventTriggered(ChannelHandlerContext ctx, Object evt) {
+                        if (evt instanceof Http2GoAwaySentEvent) {
+                            goAwayEventLatch.countDown();
```

Comment:
nit: maybe also verify we do not receive more events then what we expect here.

Reply:
I'll just stick to verifying "at least once" for now. adding a timeout to the test to verify if more are seen maybe unreliable.